### PR TITLE
Add a PR triage policy to the contributing guide.

### PR DESCRIPTION
**Pull request by amirh**

### CODE REVIEW

**Comment by amirh** [comment]
@Hixie @csells @kf6gpe

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,3 +110,18 @@ You must complete the
 You can do this online, and it only takes a minute.
 If you've never submitted code before, you must add your (or your
 organization's) name and contact info to the [AUTHORS](AUTHORS) file.
+
+### The review process
+
+Reviewing PRs often requires a non trivial amount of time, as our goal is to use maintainers' time in the most impactful way, we perform prioritization on issues and not on PRs. Issues pertaining to this repository are managed in the [flutter/flutter issue tracker and are labeled with "plugin"](https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3Aplugin+sort%3Areactions-%2B1-desc). Non trivial PRs should have an associated issue that will be used for prioritization. See the [prioritization section](https://github.com/flutter/flutter/wiki/Issue-hygiene#prioritization) in the Flutter wiki to understand how issues are prioritized.
```
[the remark this answers is not in the document]
Comment (amirh) [reply CONTRIBUTING.md]: thanks

**Comment by Hixie** [comment]
Ideally all of this would go to the wiki, as we have done for the engine and framework contributor docs. (Doesn't have to block this PR, though.)

**Comment by collinjackson** [comment]
@amirh should we land this PR or move it to the wiki as @Hixie suggested?

**Comment by amirh** [comment]
Filed https://github.com/flutter/flutter/issues/35747 to move all contents from CONTRIUBTING.md to the wiki.

Landing this for now.

**Pull request merged**